coding-standards/html-css/README.md

```diff
@@ -1,40 +1,93 @@
+- Use 4 spaces to indent stuff.
+- Write single line CSS, but keep [property grouping and order](#property-order--grouping) in mind.
 - Limit your lines to 80 characters. If you need to split up lines, bundle related properties.
 - Each selector's on its own line.
 - Media queries get their own line.
 - Values are preceded by a space.
 - Selectors are followed by a space.
-- Selectors should be lowercase
-- Sass variables should be camelCase
+- Selectors, sass variables and mixins should be _kebab-case_ (`.site-header`, `$primary-color`, `@mixin bp-min`).
+- When using a lot of recurring property values in your module, a scoped variable might come in handy (`$card-padding-small: 40px;`).
 - Use [BEM](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) writing style.
 - Shortest name possible, while still being descriptive. So not `.picture`, but `.profile-picture` or `.user-profile-picture`.
-- No abbreviations. Don't shorten 'image' to 'img' since there is no clear rule on common abbreviations.
+- No abbreviations. Don't shorten `image` to `img` since there is no clear rule on common abbreviations.
+- Avoid using 'magic numbers' or unnecessary complex solutions, when simpler solutions are available.
 
-
-### Property order
+### Property order & grouping
 
 While not enforced by linting, finding properties is easier when they're always in roughly the same order.
 
 - `Visibility (visibility, opacity, overflow)`
-- `Positioning (z-index, position, float etc.)`
-- `Box model (box-sizing, display, flexbox, width/height, margin, padding, borders etc.)`
+- `Positioning (z-index, position, top/right/bottom/left, float etc.)`
+- `Box model (box-sizing, margin, display, flexbox, width/height, padding, borders etc.)`
 - `Colors (background, color)`
-- `Fonts (font-family, line-height, text-decoration etc.)`
+- `Fonts (font-family, font-weight, line-height, text-decoration etc.)`
 - `Transitions and transforms`
 - `Various (box-shadow etc.)`
 
+We deem certain properties more important than others. When a property influences other elements, it's probably more imporant than a property which does not. Also keep in mind how the [box model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model) roughly works. With our property order, we're mostly going from the outside to the inside, following the visual representation of this model.
+
+A common example would be:
+
+```css
+.class {
+    z-index: 100; position: absolute; top: 0; left: 0;
+    display: block; width: 100%; height: 100%;
+    background: $blue; color: $white;
+}
+```
+
+If we would just append properties on the fly, we would always be looking for them, resulting in sloppy code and possible regressions since you might miss important properties:
+
+```css
+.class {
+    width: 100%; height: 100%;
+    background: $blue;
+    color: $white;
+    position: absolute; top: 0;
+    display: block; z-index: 100; left: 0;
+}
+```
```

Comment:
Ik vind het altijd wel aan te raden in dit soort code blocks een comment te plaatsen along the lines of "// Don't do this!" of iets dergelijks.
Zodat de luie artikel-scanner niet abusievelijk dit blok in zijn code plakt. 😉

Reply:
Yep! 👍